Gems/EMotionFX/Code/EMotionFX/Tools/EMotionStudio/Plugins/StandardPlugins/Source/AnimGraph/ParameterEditor/Vector3GizmoParameterEditor.h

```diff
@@ -50,10 +51,16 @@ namespace EMStudio
         AZ::Vector3 GetMinValue() const;
         AZ::Vector3 GetMaxValue() const;
 
+        void OnManipulatorMoved(const AZ::Vector3& position);
+
     private:
         AZ::Vector3 m_currentValue = AZ::Vector3::CreateZero();
         QPushButton* m_gizmoButton = nullptr;
+
+        // TODO: Remove this when we remove the opengl widget
```

Comment:
uber nit: This is arguable fine but I know people aren't super keen on adding `TODO`s in the code. Might be worth referencing a GHI instead? Not a huge deal just wanted to mention it in case